packages/playground/src/views/not_yet_implemented.vue

```diff
@@ -1,9 +1,43 @@
 <template>
-  <div>not_yet_implemented</div>
+  <img class="centered-img" src="images/404.png" />
```

Comment:
```suggestion
  <img class="centered-img" :src="baseUrl +  'images/404.png'" />
```
Please add the baseUrl before the image path

check `/packages/playground/src/components/dynamic_tabs.vue`
- line *#6*
- line *#79*